src/features/island/fruit/FruitPatch.tsx

```diff
@@ -80,145 +79,141 @@ export const FruitPatch: React.FC<Props> = ({ id }) => {
     (prev, next) =>
       HasAxes(prev, collectibles, fruit) === HasAxes(next, collectibles, fruit)
   );
-  const playing = useSelector(gameService, isPlaying);
+  const hasAxes = HasAxes(inventory, collectibles, fruit);
 
-  const displayInformation = async () => {
-    // First click show error
-    // Second click show panel with information
-    setShowInfo(true);
-    await new Promise((resolve) => setTimeout(resolve, POPOVER_TIME_MS));
-    setShowInfo(false);
-
-    infoToShow === "error" ? setInfoToShow("info") : setInfoToShow("error");
-  };
-
-  const harvestFruit = () => {
-    if (!fruit) return;
+  const plantTree = async () => {
     try {
-      const newState = gameService.send("fruit.harvested", {
+      const newState = gameService.send("fruit.planted", {
         index: id,
+        seed: selectedItem,
       });
 
       if (!newState.matches("hoarding")) {
-        harvestAudio.play();
-        setPlayAnimation(true);
+        plantAudio.play();
       }
-    } catch (e: any) {
-      displayInformation();
+    } catch {
+      undefined;
```

Comment:
What does this do?

Reply:
If they are not in the catch block, there are console errors when a non-fruit seed is planted.  Is it okay to not wrap them in catch blocks?